features/managing_order_message/register_order_note.feature

```diff
@@ -0,0 +1,28 @@
+@order_message
+Feature: Register order note
+  In order to register a note on on the order detail page
+  As an Administrator
+  I want to have an appropriate form on Order view in menu
+
+  Background:
+    Given the store operates on a single channel in "United States"
+    And the store has "VAT" tax rate of 15% for "Tools" within the "US" zone
+    And the store has a product "Screwdriver" priced at "$8.00"
+    And it belongs to "Tools" tax category
+    And the store has "DHL" shipping method with "$5.00" fee
+    And the store allows paying with "Cash on Delivery"
+    And there is a customer "sylius@mangoweb.cz" that placed an order "#00000001"
+    And the customer bought 10 "Screwdriver" products
+    And the customer "Mango Web" addressed it to "Street", "12345" "Los Angeles" in the "United States"
```

Comment:
Mango Web -> manGoweb

Reply:
DONE

Reply:
still "Mango Web"

Reply:
DONE

Reply:
it seems it needs a first name and a last name for the customer

Reply:
Ok, it has to be first name and last name... use "Johnny Appleseed"

Reply:
DONE